test/test_regex.py

```diff
@@ -37,23 +45,41 @@ def test_unstripped_known(self):
         line = '0x000055555557259c <+11340>:	addr32 call 0x55555558add0 <_Z19exportDebugifyStats>'
         call_match = self.unstrip_regex.match(line)
 
-        self.assertNotEqual(call_match, None)  # FIXME
+        self.assertIsNot(call_match, None)  # FIXME
         self.assertEqual(call_match[1], '11340')
         self.assertEqual(call_match[2], '???')
 
     def test_stripped_pic(self):
         line = '0x000055555556fd8c:	call   *0x26a16(%rip)        # 0x5555555967a8'
         call_match = self.strip_regex.match(line)
 
-        self.assertNotEqual(call_match, None)
+        self.assertIsNot(call_match, None)
         self.assertEqual(call_match[1], '55555556fd8c')
         self.assertEqual(call_match[2], '*0x26a16(%rip)        # 0x5555555967a8')  # FIXME: remove trash
 
+    @unittest.expectedFailure
+    def test_objdump_plt(self):
+        line = '1048:	e8 d3 ff ff ff       	callq  1020 <foo@plt>'
+        call_match = self.strip_regex.match(line)
+
+        self.assertIsNot(call_match, None)
+        self.assertEqual(call_match[1], '1048')
+        self.assertEqual(call_match[2], '1020 <foo@plt>')
+
+    @unittest.expectedFailure
+    def test_gdb_plt(self):
+        line = '0x0000000000001048 <bar+13>:	callq  0x1020 <foo@plt>'
+        call_match = self.strip_regex.match(line)
+
+        self.assertIsNot(call_match, None)
+        self.assertEqual(call_match[1], '1048')
+        self.assertEqual(call_match[2], 'callq  0x1020 <foo@plt>')
+
```

Comment:
Now these are called `test_gdb_*` and `test_objdump_*`. Maybe we should prefix all other tests similar way to distinguish between the tests.

Reply:
Yeah, I also thought about that.